client/dashboard/app/queries/a8c.ts

```diff
@@ -0,0 +1,8 @@
+import { fetchReaderTeams, ReaderTeam } from '../../data/reader-teams';
+
+export const isAutomatticianQuery = () => ( {
+	queryKey: [ 'me-is-automattician' ],
```

Comment:
Sorry for not being clear. I mean something like below. The benefits is we can invalidate the cache by the query keys like `[ 'me' ]` 

```js
[ 'me', 'is-automattician' ]
```